srcpkgs/bumblebee/template

```diff
@@ -10,20 +10,24 @@ configure_args="
  CONF_MODPATH_NVIDIA=/usr/lib/xorg/modules
  --with-udev-rules=/usr/lib/udev/rules.d/
  --without-pidfile"
-hostmakedepends="pkg-config help2man"
-makedepends="VirtualGL-devel glib-devel MesaLib-devel"
-depends="VirtualGL"
 conf_files="
  /etc/bumblebee/bumblebee.conf
  /etc/bumblebee/xorg.conf.nouveau
  /etc/bumblebee/xorg.conf.nvidia"
-system_groups="bumblebee"
+hostmakedepends="automake autoconf-archive pkg-config help2man"
+makedepends="VirtualGL-devel glib-devel MesaLib-devel libkmod-devel"
+depends="VirtualGL"
 short_desc="NVIDIA Optimus support for Linux through VirtualGL"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-homepage="http://www.bumblebee-project.org/"
 license="GPL-3"
```

Comment:
please use SPDX short-identifier for license=

Reply:
Fixed